ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/RedHat.yml

```diff
@@ -25,6 +25,13 @@
     - ansible_distribution_major_version == "8"
   tags: patch_update
 
+- name: Enable EPEL release for RHEL6
```

Comment:
This is doing the same thing as the task above it- just modify the conditions, as it already has the `{{ ansible_distribution_major_version }}` variable in it :-)

Reply:
Good point